documentation/manual/src/main/asciidoc/modules/infinispan.asciidoc

```diff
@@ -21,6 +21,27 @@ for a complete description of all its capabilities and configuration options,
 refer to the Infinispan project documentation at
 http://infinispan.org/documentation/[infinispan.org].
 
+=== Why use Hibernate OGM with Infinispan?
+
+Infinispan offers a lot in term of scalability and elasticity but
```

Comment:
"a lot" is vague: what are you referring to? Money?
"term" should be plural: "terms"

"Infinispan provides great scalability and elasticity features ..."